Dockerfile

```diff
@@ -71,6 +71,7 @@ WORKDIR /app
 
 # Copy virtual environment from builder
 COPY --from=builder-base ${VIRTUAL_ENV} ${VIRTUAL_ENV}
+ENV PATH="/app/.venv/bin:$PATH"
```

Comment:
Interesting. Are we building the virtual env in `docker run` or something? I'm curious about the reason behind it.

Reply:
@nam-m set up most of the Docker file, so I'm not completely sure what is happening here. However, this line was needed because, inside Docker, it did not recognize the virtual environment without it.